recipe/run_test.py

```diff
@@ -6,6 +6,9 @@
     from nose import run_exit
     from traits.etsconfig.api import ETSConfig
     import os
+    from subprocess import call
+    call("python setup.py build_ext --inplace",
```

Comment:
I see what scikit learn does different than us:
they install thingy using `python setup.py develop` which compiles and uses everything in place, so they dont need to build things twice: for global user path and for tests